### PR TITLE
Don't report user-induced errors to AppSignal

### PR DESCRIPTION
**Pull request by northeastprince**
## The Problem
Errors are getting reported that aren't actual issues on our end, like `RecordNotFound` or `InvalidCrossOriginRequest` errors.
## The Solution
Just don't report them in the first place!

### CODE REVIEW

**Review by garyhtou — APPROVED** [review]

File: config/appsignal.yml [hunk 1]
```diff
@@ -3,6 +3,18 @@ default: &defaults
...
+    - ActionDispatch::Http::MimeNegotiation::InvalidType
+    - ActionDispatch::Http::Parameters::ParseError
+    - ActionController::BadRequest
+    - ActionController::UnknownHttpMethod
+    - ActionController::MethodNotAllowed
+    - ActionController::UnknownFormat
+    - ActionController::ParameterMissing
+    - ActionController::InvalidCrossOriginRequest
+    - ActionController::InvalidAuthenticityToken
+    - ActiveRecord::RecordNotFound
```
Comment (garyhtou) [inline config/appsignal.yml]: We may want to report `RecordNotFound` in certain cases. Such as `RecordNotFound` in jobs. What are your thoughts?

**Review by garyhtou — CHANGES_REQUESTED** [review]

File: config/appsignal.yml (hunk 1 above)
Comment (northeastprince) [reply config/appsignal.yml]: It's pretty standard to not report `RecordNotFound` errors, so if it means something went wrong if it occurs in a job, I think a different/more-specific exception should be raised.

Comment (garyhtou) [reply config/appsignal.yml]: Let's report `ActiveRecord::RecordNotFound`. It can be indicative of a bug on our end

Comment (garyhtou) [inline config/appsignal.yml]:
```suggestion
```

**Review by garyhtou — CHANGES_REQUESTED** [review]

File: config/appsignal.yml (hunk 1 above)
Comment (northeastprince) [reply config/appsignal.yml]: Ok,  but then we'll have to change the notification settings as they pop up for controllers so we don't get notified for them after every deployment.

**Review by garyhtou — APPROVED** [review]

**Comment by garyhtou** [comment]
>Ok, but then we'll have to change the notification settings as they pop up for controllers so we don't get notified for them after every deployment.

Yes! Let's do that! Can you see if you're able to configure that in AppSignal?

**Pull request merged**